enterprise/fulltext-addon/src/main/java/org/neo4j/kernel/api/impl/fulltext/LuceneFulltextDocumentStructure.java

```diff
@@ -62,6 +62,14 @@ static Field encodeValueField( String propertyKey, Value value )
         return LuceneFulltextFieldEncoding.encodeField( propertyKey, value );
     }
 
+    public static Document documentForPopulation( long entityId, Map<String,Object> properties )
```

Comment:
don't think you need it as well since our standard index population know how to overcome that limitation already

Reply:
It makes no observable difference to performance to build a list of capturing lambdas instead, which we'd have to do because the call needs to know the size of the iterator up front. So I'm not sure it's worth the trouble in this case.

Reply:
it's more a question to have common way of doing things and avoid bicycles here and there that do the same thing but on their own special way

Reply:
Alright.